saleor/userprofile/i18n.py

```diff
@@ -70,13 +77,24 @@ class AddressForm(forms.ModelForm):
         ('email', 'email')
     )
 
+
     class Meta:
         model = Address
-        exclude = []
+        exclude = ['phone']
+
+    phoneprefix = ChoiceField(choices=phone_prefixes)
+    phone = forms.CharField()
+
 
     def __init__(self, *args, **kwargs):
         autocomplete_type = kwargs.pop('autocomplete_type', None)
+        phone = kwargs.pop('phone', None)
+        phoneprefix = kwargs.pop('phoneprefix', None)
         super(AddressForm, self).__init__(*args, **kwargs)
+        if self.instance.phone:
```

Comment:
Not sure why, but form does not see `phone` and `phoneprefix` provided in the way `form(instance=instance, dtaa, phone=phone`...), without those lines.

Reply:
This looks like a workaround and not a solution.